readme-prodekanus.md

```diff
@@ -15,11 +15,12 @@ For at opdatere siden med de nyeste ændring fra Github skal du gøre følgende:
 ```sh
     sudo -u tkammer -s  # start en ny shell med tkammer som bruger
     cd /home/tkammer/tkweb  # Skift til mappen med siden.
-    source venv/bin/activate  # Aktiver virtualenv.
     git pull  # Hent de nyeste ændringer fra Github.
-    pip install -U pip -r requirements.txt  # Installer og opdater alle python pakker i virtualenv. Det kan være at den skal køres flere gange.
-    python3 manage.py migrate --settings=tkweb.settings.prod  # Migrer databasen til en evt. ny model.
-    python3 manage.py collectstatic --settings=tkweb.settings.prod  # Saml statiske filer så apache kan finde dem.
+    export PIPENV_VENV_IN_PROJECT=1  # Sig til pipenv at den skal bruge .venv mappe inde i projektet
+    pipenv install --three  # Installer og opdater alle python pakker i virtualenv. Det kan være at den skal køres flere gange.
```

Comment:
Hm, hvorfor skal `pip install -U -r requirements.txt` eller `pipenv install --three` køres flere gange?

Reply:
Jeg ved ikke med `pipenv`, men `pip install -U pip -r requirements.txt` (med `-U pip`) skal køre to gange hvis `pip` selv bliver opdateret.

Reply:
Jeg kan ikke se at nogen af dem køres flere gange, kan du uddybe?

Reply:
Hvis man kører `pip install -U pip -r requirements.txt` og ens `pip`-pakke er outdated bliver `pip`-pakken og kun `pip`-pakken opdateret. Man skal derfor køre kommandoen igen for at opdatere det der er i `requirements.txt`. Det var derfor jeg skrev kommentaren for lang tid siden. Jeg tror ikke at `pipenv install` har brug for at køre flere gange, så kommentaren kan slettes.

Reply:
Ah, havde ikke læst kommentaren. Sletter den...